cmd/rook/ceph/cleanup.go

```diff
@@ -63,6 +80,19 @@ func startCleanUp(cmd *cobra.Command, args []string) error {
 		}
 		// Remove all the mon directories.
 		cleanMonDirs()
+
+		// Initialize context to execute commands
+		context := createContext()
+
+		// Update LVM config for ceph-volume lvm list to parse devices correctly
+		pvcBackedOSD, _ := strconv.ParseBool(clusterOnPVC)
+		if err := osd.UpdateLVMConfig(context, pvcBackedOSD, true); err != nil {
```

Comment:
What if we just always updated the lvm config during the cleanup? Even if we're not cleaning up LVM-based OSDs, it won't hurt anything else, right?

Reply:
Not sure to understand, that's what we do already.

Reply:
Now I don't understand my comment either. :) The LVM config is already happening always before the OSD cleanup.